back-end/config/loader/Firebase.js

```diff
@@ -0,0 +1,47 @@
+const AbstractLoader = require('./AbstractLoader');
+const Firebase = require('../../storage/Firebase');
+
+class Config extends AbstractLoader {
```

Comment:
Please change the class name :)

Reply:
Yes, I had issues with this already. It's maybe a ConfigLoader but it's actually a Config as well. Maybe the ConfigLoader should return a Config with it's load() method :)

Reply:
Fixed!